### DISCUSSION TITLE
OpenCorporates.com: is it no longer open?

### DISCUSSION THREAD

**Discussion by rufuspollock** [datasets/commons#386] [Is It Open Data?]
OpenCorporates.com founded by Chris Taggart was once a shining exemplar for Open Data - and especially an open data business. No longer seems so as of 2023.

NB: in theory the database is available under and Open Database License (see terms of use).

However, there is no way to actually access the data in bulk without paying - which contradicts the open definition and renders the value of the openness in practice an empty offer.

Aug 1 2023 announcement requires sign-up for most activities even on the website: https://knowledge.opencorporates.com/knowledge-base/website-data-access-changes/#our-terms-of-use

![image](https://github.com/datasets/awesome-data/assets/180658/8bd26452-ccb9-4167-8da9-a4c4448c8f18)

### Clearly restrict access for bulk which is what is essential to true reuse (paid only)

See https://opencorporates.com/terms-of-use/

> Our goal is to transform corporate transparency by making company information more accessible, more usable and better quality.
> 
> Our company data is made available:
> 
> As a bulk download – [contact us](https://opencorporates.com/contact/) if you wish to access the data this way;
> Via our APIs – [see here for more details](https://opencorporates.com/plug-in-our-data/); and
> Via this website.
> We prohibit (even for Permitted Users) certain activities that place excessive demand on our servers and systems.  Prohibited use includes:
> 
> Using any automated system (e.g. scraping) to extract data for any purpose other than search engine indexation;
> Extracting large volumes of data over a short period of time (note: if you require large volumes of data, consider our API or Bulk Data offerings).
> Search engines are expressly permitted to web-crawl pages on OpenCorporates in order to index content.  Accordingly, we do not use the robots exclusion protocol. The absence of a robots.txt file does not mean we permit scraping for any other purpose.
> 
> We reserve the right to:
> 
> monitor the usage of this website and the API and any other OpenCorporates’ products in order to enforce these Terms; and
> take any action we consider necessary in order to enforce our rights and protect our interests,      including but not limited to blocking or restricting access to the company data, issuing legal proceedings or pursue legal action and/or notifying regulatory or law enforcement authorities.  
> Note that the use of this website and/or the API or any company data in violation of these Terms      may also constitute a criminal offence under applicable law, including section 1 of the UK Computer Misuse Act 1990 and the US Computer Fraud and Abuse Act.

![image](https://github.com/datasets/awesome-data/assets/180658/3edd6b20-b3d5-4bd7-a6ad-f32b5f751170)

**Comment by mitar** [comment]
It also seems it does not have much activity, has old/stale data. :-(

**Reply by rufuspollock** [reply]
Thanks for sharing @mitar - interesting data point.
